core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,796 @@
-              </p>
-              <a id="license">
-                <h1 class="oppia-license-link"></h1>
-              </a>
-              <h3 [innerHTML]="'I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5_LICENSE_HEADING' | translate"></h3>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5' | translate">
-              </p>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_6' | translate">
-              </p>
-              <div class="text-center">
-                <a href="/donate" class="btn oppia-about-button oppia-about-foundation-tab-anchor" [innerHTML]="'I18N_ABOUT_PAGE_FOUNDATION_TAB_DONATE_BUTTON' | translate">
-                </a>
-                <a href="/contact" class="btn oppia-about-button oppia-about-foundation-tab-anchor" [innerHTML]="'I18N_ABOUT_PAGE_FOUNDATION_TAB_GET_INVOLVED_BUTTON' | translate">
-                </a>
-              </div>
-            </div>
-          </div>
-
-          <div class="credits oppia-about-tab-content" [ngClass]="{'oppia-about-visible-content': activeTabName === TAB_ID_CREDITS}">
-            <div class="oppia-static-card-content oppia-static-card-content-wide">
-              <h2 [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_TAB_HEADING' | translate"></h2>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_TAB_TEXT' | translate">
-              </p>
-              <div class="oppia-about-credits-letter-groups three-col">
-                <div *ngFor="let credit of allCredits">
-                  <span>{{ credit.letter }}</span>
-                  <ul>
-                    <li *ngFor="let name of credit.names">{{ name }}</li>
-                  </ul>
-                </div>
-              </div>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_TAB_TEXT_BOTTOM' | translate :{listOfNames: listOfNames}">
-              </p>
-              <p [innerHTML]="'I18N_ABOUT_PAGE_CREDITS_THANK_TRANSLATEWIKI' | translate">
-              </p>
-            </div>
+  <div class="oppia-about-page-section-two oppia-about-section">
+    <div class="oppia-about-section-two-top oppia-about-fixed-section">
+      <div class="oppia-section-two-content">
+        <h3 class="oppia-about-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_OUR_FEATURES' | translate"></h3>
+        <p class="oppia-about-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_OUR_FEATURES_CONTENT' | translate"></p>
+      </div>
+      <div class="oppia-about-icons">
+        <div class="oppia-about-features-container oppia-flex oppia-flex">
+          <div class="oppia-about-feature-description-container oppia-flex oppia-flex-col" *ngFor="let feature of features let i = index">
+            <img [src]="getStaticImageUrl(feature.imageFilename)" alt="{{feature.i18nDescription | translate}}" class="oppia-feature-icon">
+            <div class="oppia-about-feature-text" [innerHTML]="feature.i18nDescription | translate"></div>
```

Comment:
Why everywhere we are using h2/p tag for translated text but a div tag here?

Reply:
added a tag